crates/rpc/src/handlers/validator.rs

```diff
@@ -35,6 +36,12 @@ impl ValidatorData {
     }
 }
 
+#[derive(Debug, Serialize)]
+struct ValidatorBalance {
+    index: String,
+    balance: String,
```

Comment:
these should be u64, look at other structs to see how you would add the serialization to a string

Reply:
Fixed in the latest comment with `u64` as in other structs